src/Tabs.tsx

```diff
@@ -1 +1,38 @@
-// implement a component here
+import React from 'react';
+import { Tab } from './react-app-env';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div>
+      {tabs.map(tab => (
+        <div key={tab.id}>
+
+          <button
+            type="button"
+            onClick={() => {
+              onTabSelected(tab);
```

Comment:
Will be better to change selected tab only if selected tab id is not equal to current tab id